fatrace.c

```diff
@@ -645,6 +645,9 @@ parse_args (int argc, char** argv)
         switch (c) {
             case 'C':
                 option_comm = strdup (optarg);
+		if (!option_comm) {
+		    err(EXIT_FAILURE, "memory allocation failed for --command");
+		}
```

Comment:
Still misaligned. There are tabs, and presumably your editor shows them weirdly or so? Also, no need for the {} wrapping.

Reply:
I took the liberty to just fix this myself.